### PR TITLE
[Feature]: Support Font-Awesome 5

### PR DESCRIPTION
**Pull request by denniskuligin**
# I'm submitting a...

- [ ] Bug Fix
- [x] Feature
- [ ] Other (Refactoring, Added tests, Documentation, ...)

### Checklist

- [ ] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
  - A document related commit is prefixed "docs:"
- [ ] Docs have been added / updated (for bug fixes / features)

### Description

_Font Awesome pro version support_

_you can add [fontawesome] which should be equal to "fa" || "fal" (fa light) || "fas" || "far"_

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### Does this PR affects any existing issues?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by Sibiraj-S** [comment]
Hi, Thanks for the PR, It looks cool. Let me hold it for a while.

I was planning to get rid of font-awesome and allow user to include whatever icon set they need.

**Comment by denniskuligin** [comment]
> I was planning to get rid of font-awesome and allow user to include whatever icon set they need.

That's fine, but it will take a lot of time for this ( you'll need to get mapping for that, but it takes time. And allowing FA Pro wont make it harder :) (But will allow to use it with awesome icons)

**Comment by Sibiraj-S** [comment]
@denniskuligin . I get it. I am working on removing code-mirror. Once I complete that I will merge this along. I guess it will be done today or tomorrow.

**Comment by denniskuligin** [comment]
@Sibiraj-S GREAT, GREAT NEWS

**Comment by Sibiraj-S** [comment]
@denniskuligin . I need few clarifications. Are the icon prefixes is unique in fa5 for all the icons that are currently used.

Also Update your branch. Use `angular-whitespaces` for interpolations.

File: src/app/ngx-editor/ngx-editor-toolbar/ngx-editor-toolbar.component.html [hunk 1]
```diff
@@ -2,143 +2,143 @@
   <div class="ngx-toolbar-set">
     <button type="button" class="ngx-editor-button" *ngIf="canEnableToolbarOptions('bold')" (click)="triggerCommand('bold')"
       title="Bold" [disabled]="!config['enableToolbar']">
-      <i class="fa fa-bold" aria-hidden="true"></i>
+      <i class="{{fontawesome}} fa-bold" aria-hidden="true"></i>
```
Comment (Sibiraj-S) [inline src/app/ngx-editor/ngx-editor-toolbar/ngx-editor-toolbar.component.html]: Hi, use angular white spaces where ever required

**Comment by Sibiraj-S** [comment]
Closing this due to inactivity. You can open this again whenever you have time to work on it.

**Pull request closed without merging**